google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleSupportProvider.java

```diff
@@ -165,6 +128,48 @@ public static void addSupport(@NotNull AppEngineFlexibleFacet facet,
     setupDeploymentRunConfiguration(facet.getModule());
   }
 
+  /**
+   * Generates an app.yaml configuration file in the src/main/appengine directory. If an app.yaml
+   * already exists it will not overwrite the file.
+   */
+  private static void generateAppYaml(@NotNull AppEngineFlexibleFacet facet,
+      @NotNull VirtualFile[] contentRoots) {
```

Comment:
I think there's the same problem as before -- contentRoots can be empty.

Since you only need one, you could just pass a @NotNull VirtualFile moduleContentRoot